src/views/NetworkPicker.tsx

```diff
@@ -0,0 +1,149 @@
+import { useEffect } from "react";
```

Comment:
this is a conversion of NetworkPicker to TSX. I manually tested and will be adding unit tests to this in a followup PR